javascript/javascript1/week2/fullnameformal.js

```diff
@@ -0,0 +1,34 @@
+function getFullname (firstname, surname, useFormalName, gender){
+if (useFormalName) { 
+    if (gender==="man") {
+    console.log(`lord ${firstname} ${surname}`)   
```

Comment:
```suggestion
    return `Lord ${firstname} ${surname}`;
```
and similar for every `console.log`